### PR TITLE
Refactor/id endpoint

### PR DESCRIPTION
**Pull request by apete12**
What I did: Refactor api endpoint that searches by id to change id to number data type. 

Testing Notes/Extras: NA

Did this break anything?
- [X]  No
- [ ]  Yes

Type of change
- [ ]  New feature (non-breaking change which adds functionality)
- [X]  Bug fix (non-breaking change which fixes an issue)
- [ ]  Refactor(DRY-ing up/ reorganizing code, etc.)
- [ ]  Super small fix (Corrected a typo, removed a comment, etc.)
- [ ]  Added testing for: 
- [ ]  Skip all the other stuff and briefly explain the fix.

Checklist:
- [ ]  If this code needs to be tested, all tests are passing
- [X]  The code runs locally
- [X]  The code is DRY. If it's not, I tried my best
- [X]  I reviewed my code before pushing

What's next?
implement backend delete functionality

### CODE REVIEW

**Review by paulina-isabel — APPROVED** [review]
Good work.

File: server.js [hunk 1]
```diff
@@ -141,7 +140,7 @@ app.get('/api/v1/pets', (request, response) => {
 
 app.get('/api/v1/pets/:id', (request, response) => {
   const { id } = request.params;
-  const pet = app.locals.pets.find(pet => pet.id === id);
+  const pet = app.locals.pets.find(pet => pet.id === parseIn(id));
```
Comment (paulina-isabel) [inline server.js]: Is this missing a letter t? `parseIn(id)`

Comment (judy0ye) [reply server.js]: good catch Paulina!

**Review by judy0ye — APPROVED** [review]
Looks great!

**Review by aliceabarca — APPROVED** [review]
Awesome job!

**Pull request merged**